package.json

```diff
@@ -39,7 +39,7 @@
     "renderTemplates": "PACKAGE_VERSION=$(node -p \"require('./package.json').version\") ts-node --compiler-options '{\"module\": \"commonjs\", \"target\": \"ES6\" }' ./scripts/generate-oas.ts",
     "start": "HTTPS=true react-scripts start",
     "pretest": "yarn run lint && yarn run lint:style && depcheck && yarn run check:size",
-    "test": "yarn run check:types && yarn run test:unit && CI=true yarn run test:react",
+    "test": "yarn run check:types && yarn run test:unit",
```

Comment:
Is this an intentional change or a temp change to make running tests faster?
Do we still need to keep a command around where `CI=true` (maybe update test:react)?

Reply:
This is an intentional change as the react sample app doesn't work so it doesn't make sense to test it